File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentLoyalty.java
package thedarkcolour.futuremc.enchantment;

public class EnchantmentLoyalty extends EnchantmentTridentBase {
    public EnchantmentLoyalty() {
        super(Rarity.UNCOMMON, "loyalty");
    }

    @Override
    public int getMinEnchantability(int enchantmentLevel) {
        return 5 + (enchantmentLevel - 1) * 7;
    }

    @Override
    public int getMaxEnchantability(int enchantmentLevel) {
        return 50;
    }

    @Override
    public int getMaxLevel() {
        return 3;
    }
}

File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentLoyalty.kt
package thedarkcolour.futuremc.enchantment

class EnchantmentLoyalty : EnchantmentTridentBase(Rarity.UNCOMMON, "loyalty") {
    override fun getMinEnchantability(enchantmentLevel: Int): Int {
        return 5 + (enchantmentLevel - 1) * 7
    }

    override fun getMaxEnchantability(enchantmentLevel: Int): Int {
        return 50
    }

    override fun getMaxLevel(): Int {
        return 3
    }
}